.circleci/define_build_environment_variables

```diff
@@ -1,5 +1,6 @@
-#!/bin/sh -e
-safe_git_branch=${CIRCLE_BRANCH//\//-}
-short_sha="$(git rev-parse --short=7 $CIRCLE_SHA1)"
-deploy_image_and_tag="$ECR_DOCKER_IMAGE:$safe_git_branch.$short_sha"
-ECR_DEPLOY_IMAGE="$ECR_DOCKER_REGISTRY/$deploy_image_and_tag"
+#!/bin/bash -eu
+short_sha="$(git rev-parse --short=7 "$CIRCLE_SHA1")"
+export short_sha
```

Comment:
https://www.shellcheck.net/wiki/SC2155 says it's better separately:

```
$ shellcheck .circleci/define_build_environment_variables .circleci/deploy_to_kubernetes

In .circleci/define_build_environment_variables line 2:
export short_sha="$(git rev-parse --short=7 "$CIRCLE_SHA1")"
       ^-------^ SC2155: Declare and assign separately to avoid masking return values.

For more information:
  https://www.shellcheck.net/wiki/SC2155 -- Declare and assign separately to ...
```

Reply:
(My bash-fu is not strong, so I defer to shellcheck most of the time 😅)